Rubberduck.Parsing/VBA/IDeclarationFinderProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Rubberduck.Parsing.Symbols;
```

Comment:
@Hosch250 such decoupling makes it much easier to eventually *enable* tests to work with a bunch of declarations, instead of having to mock the whole entire VBE. I'm not sure about the `FooManager` naming though: none of the `Manager` classes seem to share much in common, and it's not clear why `IDeclarationFinderProvider` can't define `RefreshDeclarationFinder` (or just `Refresh`) itself.